themes/SuiteP/css/suitep-base/sidebar.scss

```diff
@@ -350,7 +255,13 @@
 .favoritesSidebar li:hover {
   background-color: $side-bar-link-color-hover;
 }
-
+.recentlyViewedSidebar .recent-links-detail .suitepicon {
```

Comment:
This selection and definitions already added in line 226

Reply:
I have removed the duplicate